ui/tokenbridge.html

```diff
@@ -246,17 +371,53 @@ <h4>From Network <span id ="network" class="badge badge-info"></span></h4>
                         receipt = await bridgeContract.methods.receiveTokens(tokenContract.options.address, amountInWei).send({from: address, gasPrice:gasPrice, gas:300000});
                         alertInfo.hide();
                         let alertSuccess = $('#alert-success');
-                        alertSuccess.html(`Token <a href="https://explorer.testnet.rsk.co/tx/${receipt.transactionHash}">transfer to the bridge successfully sent</a>, wait 10 blocks (around 5 minutes) to see the result reflected`);
+                        alertSuccess.html(`Token <a href="https://explorer.testnet.rsk.co/tx/${receipt.transactionHash}">transfer to the bridge successfully sent</a>`);
                         alertSuccess.show();
                         $('#cross').prop('disabled', false);
                         $('#spinner').hide();
 
+                        let { blockNumber } = receipt;
+                        let currentBlock = await web3.eth.getBlockNumber();
+                        let requiredConfirmations = 10;
+
+                        let countdown = $('#countdown');
+                        countdown.html('<div>Waiting for transaction to be confirmed <br /><br /><span id="time">10:00</span>')
+                        countdown.show();
+
+                        let waitingTime = 60 * 10;
```

Comment:
try to use requiredConfirmations variable instead of 10 here, so when we change to mainnet and change this number to 120 we dont have to change it in 2 places